app/api_v1/views.py

```diff
@@ -42,30 +43,30 @@ def post(self):
         depature = request_data['depature']
 
         if not Validation().valid_str_fields(_from):
```

Comment:
A normal user may not know the meaning of the word alphanumeric. It is too technical. Try something simpler.